constants.js

```diff
@@ -185,6 +185,10 @@ const constants = {
         ORPHAN_DM_TYPE: 'orphan',
     },
     maxScannedLifecycleListingEntries: 10000,
+    overheadField: [
+        'content-length',
+        'owner-id',
+    ],
```

Comment:
Could we also add `isNull` and `versionId`? (https://scality.atlassian.net/browse/RING-43323?focusedCommentId=356769)